src/webextension/background/views.js

```diff
@@ -63,6 +63,15 @@ export async function openView(name) {
 }
 
 export async function closeView(name) {
+  if (!name) {
+    console.log("closing all views");
```

Comment:
Since I just removed a bunch of our extra `console.log`s, we probably don't need this anymore. Also, I think I introduced a merge conflict here. Sorry!